.github/workflows/tag-and-release.yml

```diff
@@ -44,8 +47,15 @@ jobs:
           username: dummy
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Login to Registry1
+        uses: docker/login-action@v3
+        with:
+          registry: registry1.dso.mil
+          username: ${{ secrets.IRON_BANK_ROBOT_USERNAME }}
+          password: ${{ secrets.IRON_BANK_ROBOT_PASSWORD }}
+
       - name: Publish UDS Zarf Package
-        run: uds run -f tasks/publish.yaml packages
+        run: uds run -f tasks/publish.yaml packages --set UDS_FLAVOR=${{ matrix.flavor }}
```

Comment:
I think for the bundle publishing we will need something special for the naming, but might be wrong. I don't think uds-cli appends the flavor to tags yet?

Reply:
yeah... I wasnt sure on the state of that or planned state for the bundle publishing. Easy enough to solve for either way.

Reply:
I verified uds wasnt doing anything with the flavor. Since there isnt a way to override the tag on publish, I updated to make the registry path .../bundles/${FLAVOR}/

Reply:
While I don't love how long that makes the artifact name/path (ex: `ghcr.io/defenseunicorns/packages/uds/bundles/registry1/k3d-core-istio:0.8.0`) I think this is a simple and reasonable solve.

One way I was considering that might be a bit more in line with how zarf handles flavors would be to dynamically tack on the flavor to the tag/version during the publish step (`yq -i '.metadata.version = "0.8.0-registry1"' bundles/k3d-standard/uds-bundle.yaml` before build/publish). While I like the outcome of that...it feels pretty hacky 🤷. We could do the same thing "statically" by duplicating the bundle files (committing with the flavor in the version), but that's not very DRY.

All that to say...I'm good with the way you handled it. I've probably already thought too much about something that isn't meant to be consumed outside of dev 😅

Reply:
Yeah so I'm not a huge fan of the way i did it being different, but figured it was between that, waiting for a way for UDS to handle flavors automatically or what you suggested above. I think I slightly prefer the outcome of your suggestion because it matches how zarf gets published but as you said the yq approach is hacky and the static approach feels excessive. If anyone else has strong opinions I'd welcome them to weigh in :D

Reply:
@UncleGedd relevant discussion on uds and flavors.

Reply:
Appreciate the discussion @zachariahmiller @mjnagel ! Would one of you mind opening an issue for UDS CLI to continue the conversation and discuss proposed options?

Reply:
Opened https://github.com/defenseunicorns/uds-cli/issues/332 to have this discussion. Going to resolve this thread for now since I think we have a path forward and as mentioned before these bundles are pretty explicitly for dev use and changing the name down the road should not be a big deal.